docs/manual/dependencymanagement.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: page
-title: "Dependency Management"
-description: ""
+title: "Dependency Management for Interpreter"
```

Comment:
Doesn't current implementation work only for Apache Spark?
Reffering to
>%spark.dep

I think it worth to be specific about it here, to set right expectations for the user.